spec/docx/smoke/api_stroke_spec.rb

```diff
@@ -6,4 +6,22 @@
     docx = builder.build_and_parse('js/docx/smoke/api_stroke/get_class_type.js')
     expect(docx.elements[1].nonempty_runs.first.text).to eq('Class Type = stroke')
   end
+
+  it 'ApiStroke | ToJSON method' do
+    docx = builder.build_and_parse('js/docx/smoke/api_stroke/to_json.js')
+    expect(docx.elements.first.nonempty_runs.first.alternate_content.office2010_content.graphic.data.text_body.elements.first.character_style_array.size).to eq(2)
+    expect(docx.elements.first.nonempty_runs.first.alternate_content.office2010_content.graphic.type).to eq(:shape)
+    expect(docx.elements[1].nonempty_runs.first.text).to include('"type":"stroke"')
+    json = JSON.parse(docx.elements[1].nonempty_runs.first.text)
+    stroke = json['graphic']['spPr']['ln']
+    rgba_from_stroke = stroke['fill']['fill']['color']['color']['rgba']
+    expect(rgba_from_stroke['red']).to eq(51)
+    expect(rgba_from_stroke['green']).to eq(51)
+    expect(rgba_from_stroke['blue']).to eq(51)
+    expect(rgba_from_stroke['alpha']).to eq(255)
+    fill_type = stroke['fill']['type']
```

Comment:
I meant by the previous comment whether it is possible to get the type of Stroke object and not the type of Fill object nested in Stroke. What is the point of checking the Fill type in a Stroke test?

Reply:
Fix in a new commit.